test/_certificates/conf/extfile.conf

```diff
@@ -0,0 +1,4 @@
+[extensions]
+subjectKeyIdentifier=hash
+authorityKeyIdentifier=keyid,issuer
+keyUsage=digitalSignature,nonRepudiation
```

Comment:
I think you will also need `extendedKeyUsage` and `certificatePolicies` to be set for these certificates to work.  The details don't matter provided the OIDs you use are passed into the smart contract at deployment via `addExtendedKeyUsage` and `addCertificatePolicies`